client/main.less

```diff
@@ -144,6 +144,9 @@ body {
 .tag-rating a {
 	color: @body-text;
 }
+.tag-name a {
```

Comment:
Okay, so this is not needed, but we might need a field to display the download name, when rendering an ImageView().